site.config.json

```diff
@@ -16,10 +16,26 @@
   "explore_button_text": "🚀 Explore the Zoo",
   "subscribe_url": "https://docs.google.com/forms/d/e/1FAIpQLSeS1TV3QkTfsIXNsKGQ3bzJuAxprCkJ4VHX1DDNoaDM_mhwPg/viewform?embedded=true",
   "contribute_url": "https://github.com/CellProfiling/covid-models#how-to-contribute-new-models",
-  "footer_github": {
-    "url": "https://github.com/CellProfiling/covid-models",
-    "label": "COVID-19 Model Repository"
-  },
+  "footer": [
+    {
+      "label": "Github",
+      "tooltip": "COVID-19 Model Repository",
+      "logo": "/static/img/github.svg",
+      "url": "https://github.com/CellProfiling/covid-models"
+    },
+    {
+      "label": "SciLifeLab",
+      "tooltip": "Science for Life Laboratory",
+      "logo": "/static/img/scilifelab.png",
```

Comment:
Could you change it to a transparent version: https://www.pinclipart.com/pindetail/bxomhi_science-for-life-laboratory-logo-clipart/

Reply:
done

> Could you change it to a transparent version: https://www.pinclipart.com/pindetail/bxomhi_science-for-life-laboratory-logo-clipart/